### ISSUE TITLE
ERR_CONNECTION_REFUSED when ionic run android --livereload

### ISSUE THREAD

**Issue by ImmanuelSamuel** [Muffasa/ionic2-meteor-starter#4]
I get the following error
`GET http://localhost:3000/sockjs/info?cb=cet7qnmk04 
net::ERR_CONNECTION_REFUSED`
`GET http://192.168.0.22:8100/build/fonts/ionicons.woff2?v=3.0.0-alpha.3`
I have meteor running on localhost:3000
I also have ionic in livereload

![selection_020](https://cloud.githubusercontent.com/assets/7865018/13688392/9cc91684-e6ef-11e5-878a-caf5580939de.png)

I also cannot access it though my browser. I can do the same however if I just run Ionic serve.

**Comment by ohadgk** [comment]
As written in the readme, you must go to the meteor-core folder and run meteor.

**Comment by ImmanuelSamuel** [comment]
Yes I have meteor running also. I am able to go to localhost:3000 and see the page add stuff to mongo and can even connect to it when I run ionic serve. I just cant get it to connect if I run ionic run android --livereload.
Will try complete reinstall and let you know.

**Comment by ohadgk** [comment]
when you add the --livereload you tell ionic to serve the application from a proxy server, that might be the break. try to run it without the livereload and tel me if that works.

**Comment by ImmanuelSamuel** [comment]
Yes it works without live reload. Its working now that I am running it through a domain.
